### PR TITLE
api blueprint for deprecated version

### PR DESCRIPTION
**Pull request by honnix**
addressing #118

### CODE REVIEW

**Comment by honnix** [comment]
@fabriziodemaria PTAL

File: doc/api.apib [hunk 1]
```diff
@@ -31,40 +22,43 @@ aglio -i doc/api.apib -o api.html --theme-template triple --theme-variables stre
 
 Resources related to the Workflows registered in Styx.
 
-## Workflow [/v2/workflows/{component}/{workflow_id}]
+## Workflow [/v2/workflows/{component_id}/{workflow_id}]
 
 + Parameters
-    + component: `styx-canary` (string) - Workflow Component
-    + workflow_id: `LuigiCanary` (string) - Workflow ID
```
Comment (fabriziodemaria) [inline doc/api.apib]: Should we change the `luigi-canary-dummy:dummy` entries as well? Perhaps remove all the Luigi parts :)

File: doc/api-deprecated.apib [hunk 2]
```diff
@@ -0,0 +1,558 @@
...
+
+        {
+            "parameter": "2017-01-01T01",
+            "workflow_id": {
+                "component_id": "styx-canary",
+                "endpoint_id": "StyxCanary"
+            }
+        }
+
+# Group Global Config
```
Comment (fabriziodemaria) [inline doc/api-deprecated.apib]: Actually should we rename this `Group Styx Config` to reflect the class name as in all the other cases? Same in the file for APIs version 2. I am not sure how I came up with this "Global" name before..

**Comment by fabriziodemaria** [comment]
Minor things commented, nothing important! Feel free to merge as is 👍

File: doc/api.apib (hunk 1 above)
Comment (honnix) [inline doc/api.apib]: Will do

**Comment by honnix** [comment]
incorporate #201 to avoid a rather big merge.

**Pull request merged**